Meter.c

```diff
@@ -149,6 +163,14 @@ static void BarMeterMode_draw(Meter* this, int x, int y, int w) {
    assert(startPos + w <= RichString_sizeVal(bar));
 
    int blockSizes[10];
+#ifdef HAVE_LIBNCURSESW
+   Settings* settings = this->host->settings;
+   assert(settings->barType < ( sizeof(bars) / sizeof(wchar_t*) ));
+   assert(settings->barType >= 0);
+   const wchar_t* currBar = bars[settings->barType];
+   int barLen = (int)wcslen(currBar);
```

Comment:
I consider this function call redundant when you have declared the list of `bars` to have equal length.

Not sure if this is what you intended, but I think the bar meter characters can be defined with variable widths if you wish.

Like this:

```c
static const wchar_t* BarMeterMode_drawStyles[] = {
   L"|",
   L"#",
   L"⡀⡄⡆⡇⣇⣧⣷⣿",
   L"░▒▓█",
   L"▏▎▍▌▋▊▉█",
   L"▁▂▃▄▅▆▇█",
   L"▌█",
};
```

Reply:
The bars currently have all the same length, but the intention is to actually allow for bars to contain different number of symbols.